docs/blog/2023-03-28-timelock-on-fastnet.md

```diff
@@ -0,0 +1,150 @@
+
+## Timelock Encryption
+
+### Drand background
+
+In order to understand how the system works, we need to first recall a few things about how drand functions…
+
+First things first, let’s recall that `drand`, which stands for “**distributed randomness**” (and thus pronounced “dee-rand”), is our very own [open-source software](https://github.com/drand/drand) meant to create distributed randomness networks. These networks emit **random “beacons” at a given frequency** in a way that you never need to trust any single party in the network, and can ***verify*** that you actually have random values as long as you trust there is **never a threshold number of malicious parties** in the network, thus the security of the threshold BLS signature scheme holds.
+
+To accomplish this drand relies on the threshold BLS signature scheme and its security proof that tells us BLS signatures are indistinguishable from random ones (in the group of all possible signatures). At each epoch, the members of a drand network work together to create an aggregated BLS signature on the message **m = H(round)}**. An important point here for what comes next is that the **message is predictable** according to the target round at a time T (in other words, there is a mapping between time and round numbers, since the network operates at a fixed frequency), when we’re using drand in the so-called unchained mode that [we presented in a post last year](https://drand.love/blog/2022/02/21/multi-frequency-support-and-timelock-encryption-capabilities/#unchained-randomness-timed-encryption).
+
+You can learn more about drand and how it works under the hood in our documentation: [https://drand.love/docs/cryptography/](https://drand.love/docs/cryptography/) 
+
+### The League of Entropy
+
+The League of Entropy is a consortium of organisations that decided, back in 2019, to start running a global drand network in order to provide free, unbiased, and verifiable public randomness for anyone to use. Since 2019, the League has been growing steadily, with new members joining it regularly and thus increasing the security and trust we can have in the League. You can read more about the League on Cloudflare’s website: [https://www.cloudflare.com/leagueofentropy/](https://www.cloudflare.com/leagueofentropy/)
+
+As of March 2023, the League is constituted of over 21 drand nodes operated by 18 organisations, with a threshold of over 50%, and you can follow our blog to see when new partners are joining the League: [https://drand.love/blog/](https://drand.love/blog/)
+
+In case you were worried after reading our paper, know that whenever new members join the League, we conduct a so-called “resharing” of the League’s group secret key using verifiable secret sharing. This means that **the public key of the drand networks we are running never changes** and it is not going to cause an incompatibility issue with our timelock scheme. 
```

Comment:
```suggestion
In case you were worried after reading our paper, know that whenever new members join the League, we conduct a so-called “resharing” of the League’s group secret key using verifiable secret sharing. This means that **the public keys of the drand networks we run never change** and it is not going to cause an incompatibility issue with our timelock scheme. 
```